### PR TITLE
Friends page and notification page

### PR DESCRIPTION
**Pull request by ceciliaccwei**

### CODE REVIEW

File: src/components/notices/NoticesPage.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import React, { Component } from "react";
+import "../../styles/friends/FriendsPage.scss";
```
Comment (maharshmellow) [inline src/components/notices/NoticesPage.jsx]: Why are we using the FriendsPage.scss file here?

File: src/styles/friends/FriendItem.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
+$accent-color: #4B84F2;
```
Comment (maharshmellow) [inline src/styles/friends/FriendItem.scss]: Use the accent colour from the common file

File: src/components/friends/FriendsPage.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        <Row>
+          <Col md={2} />
+          <Col md={8}>
+            <div className="content-wrapper">
+              <div className="title">
+                <PeopleAltOutlinedIcon className="icon" />
+                <p>Friends</p>
+                <p>
+                  (
+                  {friendsList.length}
```
Comment (maharshmellow) [inline src/components/friends/FriendsPage.jsx:59]: Why are there parenthesis around this line on line 58 and 60

**Review by maharshmellow — CHANGES_REQUESTED** [review]

File: src/components/notices/NoticesPage.jsx (hunk 1 above)
Comment (ceciliaccwei) [reply src/components/notices/NoticesPage.jsx]: The friends Page and the notification page are basically using the same style. I can rename the FriendsPage.scss to Friends-NoticesPage.scss

File: src/components/friends/FriendsPage.jsx (hunk 3 above)
Comment (ceciliaccwei) [reply src/components/friends/FriendsPage.jsx:59]: based on the figma design, the top left of the page will show [icon]Friends(30). The parenthesis are in the <p> tag.

Comment (maharshmellow) [reply src/components/friends/FriendsPage.jsx:59]: Ok cool I didn’t see that 😀

**Review by maharshmellow — DISMISSED** [review]

File: src/styles/friends/FriendItem.scss (hunk 2 above)
Comment (ceciliaccwei) [reply src/styles/friends/FriendItem.scss]: done! Also changed border color and background color in Notice/Friends Page to use common file😄

**Comment by ceciliaccwei** [comment]
merged notice and friends styles to one folder notices-friends since they share the Page.scss.
changed colours in Notices and Friends to use the ones in the common file.
fixed eslint.

**Review by maharshmellow — APPROVED** [review]

**Pull request merged**